### ISSUE TITLE
Pre-training for SQUAD and fine-tuning for BIOASQ

### ISSUE THREAD

**Issue by rsanjaykamath** [dmis-lab/biobert#10]
Hello, 

Can you please explain how exactly can one replicate the state of the results on BIOASQ dataset, as you report in the paper that you pre-train on SQUAD before fine tuning on BIOASQ. I tried doing the same steps reported in this repo and found the results to be quite low if not pre-trained on SQUAD, which is normal. 

Is there any doc which explains the commands to do this? I'm sorry if I missed reading this.

Thanks

**Comment by jhyuklee** [comment]
As Wonjin said, to reproduce our results, you need to follow a few steps below.

1. Fine-tune BioBERT on SQuAD dataset (https://github.com/google-research/bert)
2. Use ckpt of step1 as initial checkpoint for fine-tunning BioASQ datasets. (Be sure to set output folder as new one)

Thanks.

**Comment by telukuntla** [comment]
Hello @jhyuklee ,
May I know what are the parameters you have used when training on Squad. like epochs and learning rate..

**Comment by jhyuklee** [comment]
#19

**Comment by sonigovind** [comment]
As I have a dataset for NER, Which is containing three features like seq, word, and tag and now I want to use the NER task so how can I use this model for my work?
